Makefile

```diff
@@ -11,7 +11,7 @@ RUNTIME_DEF_URL="https://github.com/codefresh-io/cli-v2/releases/latest/download
 ADD_CLUSTER_DEF_URL="https://github.com/codefresh-io/csdp-official/add-cluster/kustomize"
 FALLBACK_ADD_CLUSTER_DEF_URL="https://github.com/codefresh-io/cli-v2/manifests/add-cluster/kustomize"
 
-DEV_RUNTIME_DEF_URL="manifests/runtime.yaml"
+DEV_RUNTIME_DEF_URL="<csdp-official-clone>/csdp/hybrid/runtime.yaml"
```

Comment:
how is that supposed to work? this is the value that will be injected to the store in `make local`